storage/src/qmdb/current/ordered/fixed.rs

```diff
@@ -10,10 +10,15 @@ use crate::{
     },
```

Comment:
github UI doesn't allow me to target the correct line, but here's a suggestion for module docs:

```
//! An _ordered_, fixed-size value variant of a [`crate::qmdb::current`] authenticated database.
//!
//! This variant maintains the lexicographic-next active key for each active key, enabling
//! exclusion proofs (proving a key does *not* exist). Use [`super::super::unordered::fixed`]
//! if exclusion proofs are not needed.
//!
//! See [`Db`] for the main database type and [`ExclusionProof`](super::ExclusionProof) for
//! proving key non-existence.
```